### PR TITLE
Initial support for Pachyderm 2.0

### PR DESCRIPTION
**Pull request by robert-uhl**

### CODE REVIEW

File: pachyderm/templates/postgresql/service.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  labels:
+    app: postgres
+    suite: pachyderm
+  name: postgres
+  namespace: default
+spec:
+  ports:
+  - name: client-port
+    port: 5432
+    targetPort: 0
```
Comment (chainlink) [inline pachyderm/templates/postgresql/service.yaml:18]: `Targetport: 0` (should be port name)

File: pachyderm/templates/postgresql/storageclass.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    suite: pachyderm
+  name: postgres-storage-class
+parameters:
+  type: pd-ssd
+provisioner: {{ if eq .Values.pachd.storage.backend "GOOGLE" -}}
+  kubernetes.io/gce-pd
+  {{- else if eq .Values.pachd.storage.backend "AMAZON" -}}
+  kubernetes.io/aws-ebs
+  {{- else if eq .Values.pachd.storage.backend "LOCAL" -}}
+  k8s.io/minikube-hostpath
```
Comment (chainlink) [inline pachyderm/templates/postgresql/storageclass.yaml:20]: I'm ok to make the storage path adjustments we chatted about in the meeting this morning in a subsequent PR, but we should capture in an issue

**Review by chainlink — APPROVED** [review]
Approved, with a couple of small things

File: pachyderm/templates/postgresql/storageclass.yaml (hunk 2 above)
Comment (robert-uhl) [reply pachyderm/templates/postgresql/storageclass.yaml:20]: https://github.com/pachyderm/helmchart/issues/87

**Pull request merged**